README.md

```diff
@@ -179,6 +201,7 @@ styled-system can be configured with styled-components'
 
 ```jsx
 import { ThemeProvider } from 'styled-components'
+// or import { ThemeProvider } from 'glamorous'
```

Comment:
Don't go overboard with the marketing, pretty sure this would've worked with cxs months ago ;)